src/main/java/uk/gov/hmcts/ccd/data/definition/DefaultCaseDefinitionRepository.java

```diff
@@ -178,7 +178,12 @@ public CaseType getCaseType(int version, String caseTypeId) {
     }
 
     @Override
+    @Cacheable("jurisdictionListCache")
     public List<Jurisdiction> getJurisdictions(List<String> ids) {
```

Comment:
may be worth sorting the ids before passing to this method, just to make sure it generates the same cache key for the same ids that are unordered e.g. {"TEST1", "TEST2"} and  {"TEST2", "TEST1"}

Reply:
good one worth doing.

Reply:
Well, it later on turned out that doing this is causing functional failures. The reason is the return value of the method is also a List and if we treat the id lists as sets, we will have to treat the Jurisdiction lists also as sets. Otherwise, same set of Jurisdictions returned in different order is breaking the code. That's why I cancelled the implementation of this comment, or otherwise it would take further refactorings in other places of the code.